[2023-02-26T13:39:27.448+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: my_dag.inaccurate manual__2023-02-26T13:39:24.335138+00:00 [queued]>
[2023-02-26T13:39:27.455+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: my_dag.inaccurate manual__2023-02-26T13:39:24.335138+00:00 [queued]>
[2023-02-26T13:39:27.456+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-02-26T13:39:27.456+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-02-26T13:39:27.457+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-02-26T13:39:27.466+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): inaccurate> on 2023-02-26 13:39:24.335138+00:00
[2023-02-26T13:39:27.471+0000] {standard_task_runner.py:55} INFO - Started process 8848 to run task
[2023-02-26T13:39:27.474+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'inaccurate', 'manual__2023-02-26T13:39:24.335138+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmphx0gbtf1']
[2023-02-26T13:39:27.475+0000] {standard_task_runner.py:83} INFO - Job 28: Subtask inaccurate
[2023-02-26T13:39:27.531+0000] {task_command.py:388} INFO - Running <TaskInstance: my_dag.inaccurate manual__2023-02-26T13:39:24.335138+00:00 [running]> on host 8aab065f5efa
[2023-02-26T13:39:27.579+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=inaccurate
AIRFLOW_CTX_EXECUTION_DATE=2023-02-26T13:39:24.335138+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-02-26T13:39:24.335138+00:00
[2023-02-26T13:39:27.581+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-02-26T13:39:27.582+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "echo 'inaccurate'"]
[2023-02-26T13:39:27.592+0000] {subprocess.py:86} INFO - Output:
[2023-02-26T13:39:27.594+0000] {subprocess.py:93} INFO - inaccurate
[2023-02-26T13:39:27.595+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-02-26T13:39:27.616+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=inaccurate, execution_date=20230226T133924, start_date=20230226T133927, end_date=20230226T133927
[2023-02-26T13:39:27.646+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-02-26T13:39:27.660+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
